### ISSUE TITLE
A question about fine_tuning_type

### ISSUE THREAD

**Issue by Sample-design-alt** [vfeofanov/mantis#14]
Thanks for your great work! 
I set `fine_tuning_type` to `head`, but I noticed that all parameters remained trainable and the encoder part was not frozen.

<img width="729" height="299" alt="Image" src="https://github.com/user-attachments/assets/7c0d925a-fb32-4df4-b18a-7f7767f2609c" />

 After modifying the code to freeze the parameters and retesting on the UEA datasets, I observed a significant drop in performance, with the average accuracy hovering around 50%.  Could you help me analyze the reason for this?

**Comment by vfeofanov** [comment]
Hello @Sample-design-alt 
Just in case if you're still wondering about the frozen encoder performance drop, check out our updated paper https://arxiv.org/abs/2502.15637 (v2), where we show that feature normalization (standard/minmax scaling or batch normalization) is essential for linear probing (the best performance generally comes from Standard Scaling). That is why starting from the v1.1.0, the default prediction head is BatchNorm + Linear. Surprisingly, it helps even to full fine-tuning!
Finally, just want to remark that for some datasets the full fine-tuning still brings a huge improvement, so I guess there is still a room for improvement.
